mis_builder/models/mis_report_instance.py

```diff
@@ -499,18 +499,19 @@ def _compute_pivot_date(self):
         comodel_name="res.company",
         string="Company",
         default=lambda self: self.env.company,
-        required=True,
+        required=False,
     )
     multi_company = fields.Boolean(
         string="Multiple companies",
         help="Check if you wish to specify "
-        "children companies to be searched for data.",
+        "several companies to be searched for data.",
         default=False,
     )
     company_ids = fields.Many2many(
         comodel_name="res.company",
         string="Companies",
         help="Select companies for which data will be searched.",
+        domain=lambda self: [("id", "in", self.env.companies.ids)],
```

Comment:
I hesitate about this domain.

This domain hides companies that are not in the user context. In a way this is good because this is what the report will filter on. But OTOH it would be good to show the user that the report can work also on other companies if they select them in their context.

Reply:
I'll remove this domain and add `query_company_ids` to the view as `Effective companies`.